locales-uk-UA.xml

```diff
@@ -70,81 +61,72 @@
 
     <!-- ORDINALS -->
     <term name="ordinal">th</term>
-    <term name="ordinal-01">st</term>
-    <term name="ordinal-02">nd</term>
-    <term name="ordinal-03">rd</term>
-    <term name="ordinal-11">th</term>
-    <term name="ordinal-12">th</term>
-    <term name="ordinal-13">th</term>
+    <term name="ordinal-01">ий</term>
+    <term name="ordinal-02">ий</term>
+    <term name="ordinal-03">ій</term>
+    <term name="ordinal-11">ий</term>
+    <term name="ordinal-12">ий</term>
+    <term name="ordinal-13">ий</term>
```

Comment:
@vkochyn, can you tell how these ordinals are used in Ukrainian? (in particular, which numbers get which suffix)

The current set of terms used here is to accommodate English, and is probably not correct for Ukrainian. (see also http://docs.citationstyles.org/en/1.0.1/translating-locale-files.html#ordinals for background)